fdb-record-layer-lucene/src/test/java/com/apple/foundationdb/record/lucene/LuceneIndexTest.java

```diff
@@ -1723,12 +1723,15 @@ void simpleCrossPartitionQuery() {
     }
 
     static Stream<Arguments> findStartingPartitionTest() {
-        return Stream.of(true, false).map(Arguments::of);
+        Random r = ThreadLocalRandom.current();
```

Comment:
It would be better to use https://github.com/FoundationDB/fdb-record-layer/blob/f86e1dc34ae52faf04aa7cf7572bd26e49a3b4c5/fdb-extensions/src/test/java/com/apple/test/RandomizedTestUtils.java#L50
That limits injecting randomness to when it is requested, and not during PRB. This can reduce flakiness to PRB.